.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   RUST_BACKTRACE: 1
   # Some of the bindgen tests generate "deref-nullptr" warnings, see https://github.com/rust-lang/rust-bindgen/issues/1651
-  RUSTFLAGS: "--deny=warnings --allow deref-nullptr"
+  RUSTFLAGS: "--deny=warnings --allow deref-nullptr --allow renamed_and_removed_lints"
```

Comment:
Why allow this?

Reply:
If I ran:
rustup default nightly
cargo test --all

I'm getting these errors:
error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> health/src/proto/protobuf/health.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^
   |
   = note: `-D renamed-and-removed-lints` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(renamed_and_removed_lints)]`

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
 --> health/src/proto/protobuf/health_grpc.rs:7:10
  |
7 | #![allow(box_pointers)]
  |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/example/helloworld.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^
   |
   = note: `-D renamed-and-removed-lints` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(renamed_and_removed_lints)]`

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
 --> proto/src/proto/protobuf/example/helloworld_grpc.rs:7:10
  |
7 | #![allow(box_pointers)]
  |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/example/route_guide.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
 --> proto/src/proto/protobuf/example/route_guide_grpc.rs:7:10
  |
7 | #![allow(box_pointers)]
  |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/google/rpc/status.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/testing/control.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/testing/empty.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/testing/messages.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/testing/payloads.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
  --> proto/src/proto/protobuf/testing/services.rs:10:10
   |
10 | #![allow(box_pointers)]
   |          ^^^^^^^^^^^^

error: lint `box_pointers` has been removed: it does not detect other kinds of allocations, and existed only for historical reasons
 --> proto/src/proto/protobuf/testing/services_grpc.rs:7:10

So, I added that to fix them.

Reply:
I see. You can add the allow rule to proto/src/lib.rs instead of applying it globally.